app/Http/Controllers/Salesman/SalesmanOrderController.php

```diff
@@ -20,8 +20,29 @@ public function index()
 
         $orders = $uniqueOrderItemByProduct->collapse()->map(function ($product) {
             return $product->order;
-        });
+        })->unique('id');
+
+        return view('authorized.orders.index', compact('orders'));
+    }
+
+    public function edit(Order $order)
+    {
+        $order->load('orderItems');
+
+        return view('authorized.orders.edit', compact('order'));
+    }
+
+    public function update(AuthorizedOrderRequest $request, Order $order)
+    {
+        $request->validated();
+
+        $order->load('orderItems');
+        for ($i = 0; $i < $order->orderItems->count(); $i++) {
+            $order->orderItems[$i]->status = $request->status;
+            $order->orderItems[$i]->save();
+        }
+        $order->save();
 
-        return view('authoried.order.index')->with('orders', $orders);
+        return redirect(route('orders.show', $order))->with('success', 'order.update.success');
```

Comment:
```suggestion
        return redirect(route('orders.show', $order))->with('success', __('order.update.success'));
```
hình như hơi thiếu đúng ko?